src/components/Client.ts

```diff
@@ -143,27 +146,15 @@ export class Client {
   private actOnSave(context: ExtensionContext): void {
     context.subscriptions.push(
       workspace.onDidSaveTextDocument(document => {
-        if (!window.activeTextEditor) {
```

Comment:
curiosity: is the 2 checks before the document is flux stuff no longer needed? Why was it there initially if not?

Reply:
This is because we don't need to care if this document is from the active document. Any file that is saved and is a flux file should update the LSP server.